src/rgtk.rs

```diff
@@ -60,8 +60,9 @@ extern crate "cairo-sys" as cairo_ffi;
 
 extern crate glib;
 
+pub use glib::Type;
+pub use glib::{GValue, GValuePublic};
```

Comment:
It's most likely that there won't be an `rgtk` crate. Wouldn't it be weird if many crates reexported `glib`? It's debatable so shouldn't hold up this pull request anyway. You have a point about `Type` but that's modularity for you I guess..